src/Slack/SlackMessage.php

```diff
@@ -81,6 +82,8 @@ class SlackMessage implements Arrayable
      */
     protected ?bool $broadcastReply = null;
 
+    protected ?BlockBuilder $blockBuilder = null;
```

Comment:
Assuming we'll stick to this approach instead of relaxing the strictness of the `$blocks` property, this method is missing a docblock describing what this property is for/when it's used.